snap/hooks/install

```diff
@@ -42,17 +42,6 @@ for profileType in bool float int uint; do
     cp "$SNAP/config/device-virtual/res/device.virtual.$profileType.yaml" "$SNAP_DATA/config/device-virtual/res/device.virtual.$profileType.yaml"
 done
 
-# also handle java services' application.properties
-# shellcheck disable=SC2043
-for jsvc in edgex-support-rulesengine; do
-    if [ ! -f "$SNAP_DATA/config/config-seed/res/properties/$jsvc/application.properties" ]; then
-        mkdir -p "$SNAP_DATA/config/config-seed/res/properties/$jsvc"
-        cp "$SNAP/config/config-seed/res/properties/$jsvc/application.properties" "$SNAP_DATA/config/config-seed/res/properties/$jsvc/application.properties"
-        # also replace SNAP_DATA and SNAP_COMMON in the application files
-        sed -i -e "s@\$SNAP_COMMON@$SNAP_COMMON@g" -e "s@\$SNAP_DATA@$SNAP_DATA_CURRENT@g" "$SNAP_DATA/config/config-seed/res/properties/$jsvc/application.properties"
-    fi
-done
-
 # create support-rulesengine directories for templates/rules
```

Comment:
No removal of the rules-engine yet. I think we'll have over lap for one release.

Reply:
Really, I thought we were finally removing support-rulesengine for this release?

If that's true (which will make me very sad), then you need to restore the rules-engine related code in the in the install hook that your PR removes, see:

`# also handle java services' application.properties`

...and the rules-engine statements you removed from the edgex-go part in snapcraft.yaml:

```
# handle application.properties for the remaining java services
      cat "./cmd/config-seed/res/properties/edgex-support-rulesengine/application.properties" | \
        sed -e s:edgex/rules:\$SNAP_DATA/support-rulesengine/rules: \
            -e s:edgex/templates:\$SNAP_DATA/support-rulesengine/templates: > \
       "$SNAPCRAFT_PART_INSTALL/config/config-seed/res/properties/edgex-support-rulesengine/application.properties"
```

Reply:
@tonyespy,  This relies on config seed which has been removed. I think @jim was going to potentially address this in the Rules Engine if EMQ X Kuiper doesn't pan out.

Reply:
@tonyespy , Jim & Akram already modified Rules Engine to operate just off local config, ino consul. So we are good to go as it is working fine in my testing. :-)

Reply:
@lenny-intel To add some context and clarify, that work (i.e. modifying the properties of the Rules Engine to obtain its configuration `locally`, and not going to `Consul`) is underway, per [Issue-68](https://github.com/edgexfoundry/support-rulesengine/issues/68). As part of that work, I'm also updating the `application.properties` (of Rules Engine) so that it is set to the current `config-seed` properties. I'm testing that locally, both in native DEV and Dockerized, before I submit a `PR`. I will watch your work (per `PR-2445`) closely as you go about testing for the removal of config seed, etc.